GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -30,11 +30,8 @@
 /** A library data key this class uses to track batchIDs. */
 static NSString *const gBatchIDCounterKey = @"GDTCORFlatFileStorageBatchIDCounter";
 
-/** A library data key this classes uses to track batches and their associated eventIDs. */
-static NSString *const gBatchesToEventsIDsKey = @"GDTCORFlatFileStorageBatchIDCounter";
-
 /** The separator used between metadata elements in filenames. */
-static NSString *const kMetadataSeparator = @"-";
+static NSString *const kMetadataSeparator = @"|";
```

Comment:
Why change this back?

Reply:
"-" symbol is a part of UUID, so I had to change the separator. Now I think I can replace "-" in the eventID itself if needed. Do you remember issues with "|"?

Reply:
In some of the reading I did, some filesystems were sensitive to use of | as a file or directory name. Also, our % escaping only allowed alphanum and "-", make sure to update that in GDTCOREventSelector.

Reply:
Ah, ok, I'll revert the separator back and modify eventID itself. Thanks!

Reply:
Done